src/_guides/whats-new.md

```diff
@@ -17,6 +17,144 @@ and follow the [Dart blog][].
 [dart-announce]: https://groups.google.com/a/dartlang.org/d/forum/announce
 [Dart blog]: https://medium.com/dartlang
 
+## TODO DATE: 2.17 release
+
+This section lists notable changes made from February 4, 2022,
+through TODO DATE.
+For details about the 2.17 release,
+see [Dart 2.17: BLOG POST TODO][].
+
+[Dart 2.17: BLOG POST TODO]: https://medium.com/dartlang/TODO
+
+### Docs updated or added to dart.dev
+{: .no_toc}
+
+In addition to bug fixes and incremental improvements,
+we made the following changes to this site:
+
+* Introduced the [Learning Dart as a JavaScript developer][] guide
+  which aims to leverage your JavaScript programming knowledge
+  when learning Dart.
+* Documented the features and changes introduced in Dart 2.17:
+  * Updated the [Named parameters][] section of the language tour
+    to reflect support for specifying named arguments anywhere.
+  * Added documentation for [super-initializer parameters][].
+  * Expanded the [Enumerated types][] section of the language tour
+    and documented [enhanced enums][].
+  * Documented support for [signing][] macOS and Windows executables
+    compiled with `dart compile exe`.
+  * Updated the templates supported by [`dart create`][]
+    to their new, standardized names.
+* Accounted for changes to the [pub.dev site][] and the [pub tool][].
+  * Listed vendors offering [Dart package repositories as a service][].
+  * Removed documentation for the now discontinued `dart pub uploader` command.
+  * Expanded the documentation for managing pub project [uploaders][].
+  * Removed most mentions to the deprecated `.packages` file,
+    pointing instead to its `.dart_tool/package_config.json` replacement.
+* Updated the documentation for updating and installing Dart:
+  * Documented how to switch between Dart versions
+    with Homebrew within the macOS [install instructions][get-dart-install].
+  * Updated the linux [installation instructions][get-dart-install]
+    to use [SecureApt][] and follow the latest best practices.
+  * Added support for downloading experimental, Linux RISC-V (RV64GC) builds
+    from the [Dart SDK archive][].
+* Continued work to improve and update documentation
+  of the [unified `dart` tool][dart-tool]:
+  * Expanded documentation about the functionality of the [`dart fix`][] tool.
+  * Adjusted the guidelines and documentation for the [`dart doc`][] tool
+    to match its functionality and underlying behavior.
+  * Added furthered documentation and samples of [`dart compile js`][].
+  * Removed mentions of the old standalone tools.
```

Comment:
```suggestion
  * Removed mentions of the deprecated standalone tools.
```

Reply:
The mentions removed are of tools now completely removed. Is deprecated still fine here or should I say something else?